### ISSUE TITLE
Add. 14,636, foll. 1–56 | https://bl.vuexistapps.us/manuscript/396 (Part1)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#1065]
### Problems with Manuscript Overview

- Date is missing. 
- Please see the note below about the possibility of multiple hands. 

### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [X] Missing content

### Problems with Physical Description

- Phys. Des. only has Hand 1. However, additions 1 & 2 inform that a couple of notes were written by other hands in Arabic and Syriac. So, perhaps Hand 1 and Hand 2 should also be added. 

### Categories of Physical Description Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [X] Missing content

**Comment by mikey-tang16** [comment]
Resolved: 
- Arabic hand added

Question
Should I add another syriac hand for addition 2? @wlpotter

**Comment by wlpotter** [comment]
Yes, please add a "See additions" hand for unspecified Syriac script

**Comment by mikey-tang16** [comment]
Syriac hand added for additions
